docs/clusterbot.md

```diff
@@ -0,0 +1,53 @@
+How to use cluster-bot builds with dev-scripts
+==============================================
+
+One of the cool features of cluster-bot is that it can build a release image
+based on one or more unmerged pull requests.  The image can then be plugged into
+dev-scripts, and a new cluster created with that image.
+
+First, have cluster-bot build your image with something like:
+
+    build openshift/cluster-api-provider-baremetal#166
```

Comment:
The first time I had to do this I remember being very confused because I was missing a vital piece of context: that cluster-bot is a Slack bot that you interact with by sending it PMs.